### ISSUE TITLE
Add Support for Future Posting

### ISSUE THREAD

**Issue by jasongaylord** [cloudscribe/cloudscribe.MetaWeblog#4]
Metaweblog supports posting a datetime object of the blog post itself. Support should be added to allow for this.

**Comment by joeaudette** [comment]
Would be great if you want to implement that and make a pull request.

When I implemented this I found it difficult to find good documentation about metaweblog api. I added config settings so I could dump the incoming xml post to disk to be able to see the contents and write code to parse it. There are a few example files in the testfiles folder. The new post xml in the samplefiles folder doesn't seem to have a date but maybe the posted content depends to some extent on the capabilities defined in the wlwmanifest.xml
If there is a way to light up more features in OLW that would be great.
